tsdb/index/postings.go

```diff
@@ -115,6 +115,8 @@ type PostingsStats struct {
 	CardinalityLabelStats   []Stat
 	LabelValueStats         []Stat
 	LabelValuePairsStats    []Stat
+	// TODO: Currently this is only counted for Head block. It will need to include persistent blocks in the future.
```

Comment:
My bad. I looked through the code now and apparently the stats are only for Head and we never intended to add it to persistent blocks yet. So this comment is not required. Good to merge after removing this comment.